charts/lakefs/templates/dev-postgresql.yaml

```diff
@@ -1,53 +1,47 @@
 {{- if .Values.useDevPostgres }}
+{{- if and (.Values.fluffy).enabled (.Values.fluffy.rbac).enabled }}
 
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: postgres-secret
-  labels:
-    app: postgres
-data:
-  POSTGRES_DB: postgres
-  POSTGRES_USER: lakefs
-  POSTGRES_PASSWORD: lakefs
-  MAX_CONNECTIONS: "1000"
----
 apiVersion: apps/v1
-kind: Deployment # Create a deployment
+kind: Deployment
 metadata:
-  name: postgres # Set the name of the deployment
+  name: postgres
 spec:
-  replicas: 1 # Set 3 deployment replicas
+  replicas: 1
   selector:
     matchLabels:
-      app: postgres
+      app: postgres-lakefs
   template:
     metadata:
       labels:
-        app: postgres
+        app: postgres-lakefs
     spec:
       containers:
         - name: postgres
-          args: ["-c", "max_connections=1000"]
-          image: postgres:11 # Docker image
+          args: ["-c", "max_connections=50"]
```

Comment:
I prefer not to have this one set - the default is 100 and it should be enough for both lakefs and fluffy.